x-pack/legacy/plugins/apm/public/components/app/ServiceMap/LicensePrompt.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import {
+  EuiEmptyPrompt,
+  EuiButton,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useKibanaUrl } from '../../../hooks/useKibanaUrl';
+
+export function LicensePrompt() {
```

Comment:
Perhaps call it "PlatinumLicensePrompt" Since we already have another prompt that ensures the user has a license at all.

Reply:
Updated in 44d95685c5.